infra/monitor/KalturaMonitorClient.php

```diff
@@ -805,4 +805,40 @@ public static function sendErrorEvent($errorCode)
 		
 		self::writeDeferredEvent($data);
 	}
+	
+	public static function checkApiRateLimit()
+	{
+		$uri = isset($_SERVER["REQUEST_URI"]) ? $_SERVER["REQUEST_URI"] : '';
+		if(!$uri)
+		{
+			return;
+		}
+		
+		preg_match('#/p/([^/]+)/sp/([^/]+)#', $uri, $matches);
+		if(!$matches || !count($matches))
+		{
+			return;
+		}
+		
+		$partner_id = isset($matches[1]) ? $matches[1] : (isset($matches[2]) ? $matches[2]/100 : null);
+		if(!$partner_id)
```

Comment:
Is it possible to get manifest request on partner 0?

Reply:
Should not happen.